src/cljs/rems/actions/approve_reject.cljs

```diff
@@ -18,33 +16,21 @@
 (rf/reg-event-fx
  ::send-approve
  (fn [_ [_ {:keys [application-id comment on-finished]}]]
-   (let [description [text :t.actions/approve]]
-     (post! "/api/applications/approve"
-            {:params {:application-id application-id
-                      :comment comment}
-             :handler (flash-message/default-success-handler
-                       :actions
-                       description
-                       (fn [_]
-                         (collapse-action-form action-form-id)
-                         (on-finished)))
-             :error-handler (flash-message/default-error-handler :actions description)}))
+   (command! "approve" {:application-id application-id
+                        :comment comment}
+             {:description [text :t.actions/approve]
+              :collapse action-form-id
```

Comment:
should we just compose the collapse into on-finished here?

Reply:
Maybe not. The `command!` is anyways very specific to actions.